frontend/public/extend/devconsole/components/import/import-validation-utils.ts

```diff
@@ -37,6 +39,26 @@ export const validationSchema = yup.object().shape({
         message: 'This field cannot be empty',
       }),
   }),
+  route: yup.object().shape({
+    secure: yup.boolean(),
+    tls: yup.object().when('secure', {
+      is: true,
+      then: yup.object({
+        termination: yup.string().required('Please select a termination type'),
+      }),
+    }),
+    hostname: yup
+      .string()
+      .matches(hostnameRegex, {
+        message:
+          'Hostname must consist of lower-case letters, numbers, periods, and hyphens. It must start and end with a letter or number.',
+        excludeEmptyString: true,
+      })
+      .max(253, 'Can\'t be longer than 253 characters.'),
```

Comment:
For consistency with required message.
```suggestion
      .max(253, 'Cannot be longer than 253 characters.'),
```

Reply:
Done